buy-shirt.spec.ts

```diff
@@ -0,0 +1,19 @@
+describe('Buy a t-shirt', () => {
+
+  it('then the t-shirt should be bought', () => {
+    cy.visit('http://automationpractice.com/')
+    cy.get('#block_top_menu > ul > li:nth-child(3) > a').click()
+    cy.get('#center_column a.button.ajax_add_to_cart_button.btn.btn-default').click()
+    cy.get('[style*="display: block;"] .button-container > a').click()
+    cy.get('.cart_navigation span').click()
+
+    cy.get('#email').type('aperdomobo@gmail.com')
+    cy.get('#passwd').type('WorkshopProtractor')
+
+    // Debes completar la prueba ...
```

Comment:
In this step the idea is to complete the missing steps to have a successful execution, check message on slack:
https://psl-techtalks.slack.com/archives/C01QA81V29F/p1616800059027300

![image](https://user-images.githubusercontent.com/10209180/112738375-645dc080-8f30-11eb-8241-8719e1f2115e.png)